benchmark/BENCHMARK.md

```diff
@@ -38,20 +38,19 @@ Some scenarios (`CreateDeleteClients`, and `CrawlUsers`, `CreateRealms`, `Create
    * assign the roles `manage-clients` and `view-users`
 * the client secret to be passed to the tests can be copied from the `Credentials` tab
 
-#### Scenario `keycloak.scenario.admin.CreateRealms`
+#### Scenarii `keycloak.scenario.admin.CreateRealms` and `keycloak.scenario.admin.CreateDeleteRealms`
```

Comment:
please update `Scenarii` to `Scenarios` for better readability

Reply:
+1 for "scenarios". [Wikitionary](https://en.wiktionary.org/wiki/scenarii) names this as a "([nonstandard](https://en.wiktionary.org/wiki/Appendix:Glossary#nonstandard), [rare](https://en.wiktionary.org/wiki/Appendix:Glossary#rare)) [plural](https://en.wiktionary.org/wiki/Appendix:Glossary#plural_number) of [scenario](https://en.wiktionary.org/wiki/scenario#English)".

As most of the visitors to this repository are non-native English speakers, I'd like to keep wording as main-stream as possible.

Reply:
Done